docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,491 @@
+import Typography from '@mui/material/Typography';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+import KeyboardArrowUpRounded from '@mui/icons-material/KeyboardArrowUpRounded';
+import KeyboardArrowDownRounded from '@mui/icons-material/KeyboardArrowDownRounded';
+import ArrowDropDown from '@mui/icons-material/ArrowDropDown';
+import Link from 'docs/src/modules/components/Link';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import AppFooter from 'docs/src/layouts/AppFooter';
+import { MuiStats } from 'docs/src/components/home/Testimonials';
+import GradientText from 'docs/src/components/typography/GradientText';
+import { brandingDarkTheme } from 'docs/src/modules/brandingTheme';
+import IconImage from 'docs/src/components/icon/IconImage';
+import BrandingProvider from 'docs/src/BrandingProvider';
+
+const Widget = ({
+  title,
+  children,
+  url,
+  linkText,
+}: {
+  title: string;
+  children: React.ReactNode;
+  url: string;
+  linkText?: string;
+}) => {
+  return (
+    <Paper
+      variant="outlined"
+      sx={{
+        height: '100%',
+        p: 2,
+        border: '2px dashed',
+        borderColor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.400' : 'grey.400'),
+      }}
+    >
+      <Typography component="div" variant="body2" fontWeight="bold" sx={{ mb: 1 }}>
+        {title}
+      </Typography>
+      {children}
+      <Link href={url} variant="body2">
+        {linkText || 'Learn more'} <KeyboardArrowRightRounded fontSize="small" sx={{ mt: '1px' }} />
+      </Link>
+    </Paper>
+  );
+};
+
+const Question = ({
+  question,
+  children,
+  defaultShowAnswer = false,
+}: {
+  question: string;
+  children: React.ReactNode;
+  defaultShowAnswer?: boolean;
+}) => {
+  const [showAnswer, setShowAnswer] = React.useState(defaultShowAnswer);
+  return (
+    <Paper variant="outlined" sx={{ height: '100%', p: 2 }}>
+      <div style={{ width: '100%', display: 'flex', justifyContent: 'space-between' }}>
+        <Typography component="div" variant="body2" fontWeight="bold">
+          {question}
+        </Typography>
+        <span
+          role="button"
+          onClick={() => setShowAnswer(!showAnswer)}
+          onKeyDown={() => setShowAnswer(!showAnswer)}
+          tabIndex={0}
+        >
+          {showAnswer ? (
+            <KeyboardArrowUpRounded color="primary" fontSize="small" sx={{ mt: '1px' }} />
+          ) : (
+            <KeyboardArrowDownRounded color="primary" fontSize="small" sx={{ mt: '1px' }} />
+          )}
+        </span>
+      </div>
+      {showAnswer && children}
+    </Paper>
+  );
+};
+
+const Role = ({
+  title,
+  description,
+  url,
+}: {
+  title: string;
+  description: string;
+  url?: string;
+}) => (
+  <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+    <div>
+      <Typography variant="body2" color="text.primary" fontWeight={600} sx={{ my: 1 }}>
```

Comment:
How about we use it as `body1` variant? Then both title and description will be equal.